src/stats/show_stats.py

```diff
@@ -113,12 +131,24 @@ def print_percentages_above_graph_bars(graph, data):
         i += 1
 
 
-async def send_graph_to_channel(ctx, stat_id, stat_title):
+async def send_graph_to_channel(ctx, url, stat_title):
     graph_embed = discord.Embed(title=stat_title, colour=discord.Colour(0xFFFF00))
-    graph_embed.set_image(url=upload_graph_to_imgur(stat_id))
+    graph_embed.set_image(url=url)
     await ctx.channel.send(embed=graph_embed)
 
 
-def upload_graph_to_imgur(stat_id):
-    uploaded_image = imgur.upload_image(f"{stat_id}.png")
-    return uploaded_image.link
+async def upload_graph_to_imgur_or_none(ctx, stat_id):
+    try:
+        uploaded_image = imgur.upload_image(f"{stat_id}.png")
+        return uploaded_image.link
+    # Imgur occasionally responds with 403 because their servers are overloaded
+    # there is nothing to do (except buy premium)
+    except HTTPError:
+        await ctx.channel.send("Server preťažený, skús prosím neskôr.")
```

Comment:
This seems to me like an unexpected side-effect of the function. Maybe we can just return the `None` and send the message in callee. Yes we kind of lose the information about the type of the error, but we can simply always send the same messages no matter what the error is.